v1/accountapplications/accountapplications.gunk

```diff
@@ -0,0 +1,285 @@
+	ProfileID   string `pb:"3" json:"profile_id"`
+}
+
+// CreateAccountApplicationResponse is a response message for create account application
+type CreateAccountApplicationResponse struct {
+	AccountApplicationID string                   `pb:"1" json:"account_application_id"`
+	ProductCode          string                   `pb:"2" json:"product_code"`
+	User                 User                     `pb:"3" json:"user"`
+	CustomerProfile      types.Profile            `pb:"4" json:"customer_profile"`
+	DateOfApplication    time.Time                `pb:"5" json:"date_of_application"`
+	Status               AccountApplicationStatus `pb:"6" json:"status"`
+}
+
+// Structure of credit rating information
+type CreditRating struct {
+	Rating string `pb:"1" json:"rating"`
+	Source string `pb:"2" json:"source"`
+}
+
+// GetAccountApplicationpRequest is the request envelope to get an account application by its identifier.
+type GetAccountApplicationRequest struct {
+	ID string `pb:"1" json:"account_application_id"`
+}
+
+// GetAccountApplicationsRequest is the request envelope to get a list of filtered account applications.
+type GetAccountApplicationsRequest struct {
+	// NextStartingIndex is a cursor for pagination.
+	NextStartingIndex string `pb:"1" json:"next_starting_index"`
+}
+
+// GetAccountApplicationsResponse wraps the list of account applications.
+type GetAccountApplicationsResponse struct {
+	// Result is a list containing up to 20 Accounts.
+	Result []AccountApplication `pb:"1" json:"result"`
+
+	// HasMore indicates if there are more accounts available.
+	HasMore bool `pb:"2" json:"has_more"`
+}
+
+// UpdateAccountApplicationStatusRequest is the request envelope to update an account application status.
+type UpdateAccountApplicationStatusRequest struct {
+	// ID is the unique identifier of the account application to update.
+	ID string `pb:"1" json:"account_application_id"`
+
+	// Status of the account application
+	Status AccountApplicationStatus `pb:"2" json:"status"`
+}
+
+// AccountApplicationService provides Account application related operations.
+type AccountApplicationService interface {
+
+	// CreateAccountApplication creates a new account application
+	//
+	// +gunk http.Match{
+	//         Method: "POST",
+	//         Path:   "/v1/account-applications",
+	//         Body:   "*",
+	// }
+	// +gunk openapiv2.Operation{
+	//         Tags:        []string{"AccountApplication"},
+	//         Description: "Creates a new account application",
+	//         Summary:     "Create an account application",
+	//         Responses: map[string]openapiv2.Response{
+	//                 "201": openapiv2.Response{
+	//                         Description: "Account application created successfully.",
+	//                         Schema: openapiv2.Schema{
+	//                                 JSONSchema: openapiv2.JSONSchema{
+	//                                         Ref: "#/definitions/account-applicationsCreateAccountApplicationResponse",
```

Comment:
question, is this supposed to have the dash (account-applications instead of accountapplications)?

Reply:
I was using the dash before and missed to update here, also gunk doesn't like the package name with dash when I try to generate stubs.